Spatial_sorting/include/CGAL/Hilbert_sort_median_3.h

```diff
@@ -83,8 +83,14 @@ struct Hilbert_cmp_3<K,2,false>
 
 } // namespace internal
 
+#ifdef __clang__
+#define CGAL_VISIBILITY_MACRO __attribute__ ((visibility ("hidden")))
+#else
+#define CGAL_VISIBILITY_MACRO __attribute__ ((visibility ("default")))
+#endif
```

Comment:
That cannot compile with non-clang compiler that do not understand the syntax `__attribute__ ((visibility ("default")))`, like MSVC.

Reply:
You are right, I missed that.

Reply:
Why do you need a non-empty macro in the else  case if it is only a clang issue?

Reply:
I do not. My GitHub action failed when I did not have the else statement and I copy-pasted the default version, but I cannot do that since it is not going to work anywhere but for clang. It should be an empty macro.